app/screens/permalink/permalink.tsx

```diff
@@ -0,0 +1,222 @@
+    },
+    footer: {
+        alignItems: 'center',
+        justifyContent: 'center',
+        backgroundColor: theme.buttonBg,
+        flexDirection: 'row',
+        height: 43,
+        paddingRight: 16,
+        width: '100%',
+    },
+    jump: {
+        color: theme.buttonColor,
+        fontSize: 15,
+        fontWeight: '600',
+        textAlignVertical: 'center',
+    },
+    errorContainer: {
+        alignItems: 'center',
+        justifyContent: 'center',
+        padding: 15,
+    },
+    errorText: {
+        color: changeOpacity(theme.centerChannelColor, 0.4),
+        fontSize: 15,
+    },
+    archiveIcon: {
+        color: theme.centerChannelColor,
+        fontSize: 16,
+        paddingRight: 20,
+    },
+}));
+
+function Permalink({channel, postId, currentUsername}: Props) {
+    const [posts, setPosts] = useState<PostModel[]>([]);
+    const [loading, setLoading] = useState(true);
+    const theme = useTheme();
+    const serverUrl = useServerUrl();
+
+    const style = getStyleSheet(theme);
+
+    useEffect(() => {
+        (async () => {
+            const data = await fetchPostsAround(serverUrl, channel.id, postId);
+            if (data?.posts) {
+                setLoading(false);
+                setPosts(data.posts);
+            }
+        })();
+    }, []);
+
+    const handleClose = () => {
+        dismissModal();
+        closePermalink();
+    };
+
+    useEffect(() => {
+        const listener = BackHandler.addEventListener('hardwareBackPress', () => {
+            handleClose();
+            return true;
+        });
+
+        return () => {
+            listener.remove();
+        };
+    });
+
+    const handlePress = () => {
+        return null;
+    };
+
+    return (
+        <SafeAreaView
+            style={style.container}
+        >
+            <Animated.View style={style.wrapper}>
+                <View style={style.header}>
+                    <TouchableOpacity
+                        style={style.close}
+                        onPress={handleClose}
+                    >
+                        <CompassIcon
+                            name='close'
+                            size={20}
+                            color={theme.centerChannelColor}
+                        />
+                    </TouchableOpacity>
+                    <View style={style.titleContainer}>
+                        <Text
+                            ellipsizeMode='tail'
+                            numberOfLines={1}
+                            style={style.title}
+                        >
+                            {channel.displayName}
+                        </Text>
+                    </View>
+                </View>
+                <View style={style.dividerContainer}>
+                    <View style={style.divider}/>
+                </View>
+                {loading ? (
+                    <View style={style.loading}>
+                        <ActivityIndicator
```

Comment:
Just double checking: are we not using the Loading spinner component with the lottie animation?

Reply:
same question